src/EnergyPlus/MixedAir.cc

```diff
@@ -2626,9 +2614,8 @@ void InitOAController(EnergyPlusData &state, int const OAControllerNum, bool con
     }
 
     if (state.dataGlobal->BeginEnvrnFlag && state.dataMixedAir->OAControllerMyEnvrnFlag(OAControllerNum)) {
-        Real64 RhoAirStdInit = state.dataEnvrn->StdRhoAir;
-        thisOAController.MinOAMassFlowRate = thisOAController.MinOA * RhoAirStdInit;
-        thisOAController.MaxOAMassFlowRate = thisOAController.MaxOA * RhoAirStdInit;
+        thisOAController.MinOAMassFlowRate = thisOAController.MinOA * state.dataEnvrn->StdRhoAir;
+        thisOAController.MaxOAMassFlowRate = thisOAController.MaxOA * state.dataEnvrn->StdRhoAir;
```

Comment:
I'm thinking the local was good but not exactly sure.

Reply:
You're right, I mis-applied this rule from [wiki/Local-Shortcut-Variables](https://github.com/NREL/EnergyPlus/wiki/Local-Shortcut-Variables%3A-When-to-Use-a-Value-Copy-and-When-to-Use-a-Reference#the-rules)
"A local shortcut to a scalar should be a value copy unless you need to write into the scalar, in which case don't use a shortcut at all because it's confusing!"

I'll revert that.